AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Configuration/DataStoreConfiguration+Helper.swift

```diff
@@ -46,10 +81,29 @@ extension DataStoreConfiguration {
                                       syncExpressions: syncExpressions,
                                       authModeStrategy: authModeStrategy)
     }
-
+    #endif
+    
+    #if os(watchOS)
+    /// The default configuration.
+    public static var subscriptionsDisabled: DataStoreConfiguration {
+        .custom(disableSubscriptions: { false })
+    }
+    #else
     /// The default configuration.
     public static var `default`: DataStoreConfiguration {
         .custom()
     }
-
+    #endif
+    
+    #if os(watchOS)
+    /// Internal method for testing
+    static func testDefault(disableSubscriptions: @escaping () -> Bool = { false }) -> DataStoreConfiguration {
```

Comment:
I'm not a fan of the name, but I also can't think of anything other than `internalDefault`, which I equally dislike 😅